.travis.yml

```diff
@@ -12,5 +12,6 @@ before_script:
   - docker-compose -f docker-compose.yml -f docker-compose.ci.yml up --build -d
 
 script:
-  - docker exec -it cvat /bin/bash -c 'tests/node_modules/.bin/karma start tests/karma.conf.js'
+  - docker exec -it cvat /bin/bash -c 'python3 manage.py test --settings cvat.settings.testing cvat/apps/engine && 
```

Comment:
@EvgenyShashkin , do we really need --settings cvat.settings.testing here? I see that below we specify DJANGO_CONFIGURATION. I hope it should be enough. Also it is not good to close an issue before a PR which fixes it is merged. You can specify in description of PR `Resolved #...` and it will be closed automatically after the PR is merged. Could you please follow the approach?

Another problem I don't see that travis was run for the patch. Could you please explain why?